docs/README.md

```diff
@@ -1,123 +1,114 @@
 > 📢 Use this project, [contribute](https://github.com/vtex-apps/recommendation-shelf) to it, or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-# Product Recommendation
+# Recommendation Shelf
 
-> ⚠️ **Warning**
->
-> Hello! Thanks for checking out this app. It's currently under development and research by the VTEX Search and Personalization team. During this phase, the app should only be installed on selected accounts. If you're not in contact with the VTEX Search and Personalization team, installing this app on your workspace will not have any effect. If you have any questions, please contact [our Support](https://support.vtex.com/hc/en-us/requests).
+> ⚠️ This app is currently under development and research by the VTEX Search and Personalization team. During this phase, it should only be installed on selected accounts participating in the closed beta phase of the [Product Recommendations](https://help.vtex.com/en/tutorial/product-recommendations-beta--2QIexbD2FSXBxELUnFtg7g) feature. For other accounts, installing this app will not have any effect. If you want to adopt this feature for your business, please contact [our Support](https://support.vtex.com/hc/en-us/requests).
 
-## Description
-
-The VTEX Recommendation Shelf app is a store component that displays a collection of products using recommendation strategies.
+The Recommendation Shelf app is a store component that displays a collection of products using recommendation strategies.
 
 ![recommendation-shelf](https://raw.githubusercontent.com/vtex-apps/recommendation-shelf/refs/heads/master/docs/shelf.png)
```

Comment:
Hey @julia-rabello  is there a better way to display image in the docs? Here I'm pointing to a file directly from github, but I'm not  sure if we have other way of doin it....

Reply:
Hi @salesfelipe, this way looks alright, we usually point for files directly from Github to make maintenance easier.

For improved performance, it would be best to use a [jsDelivr](https://www.jsdelivr.com/?docs=gh) route in this format: `https://cdn.jsdelivr.net/gh/user/repo@version/file`. This one would be `https://cdn.jsdelivr.net/gh/vtex-apps/recommendation-shelf@master/docs/shelf.png`.

Reply:
Amazing, thanks for the clarification 🙏